docs/blog/_posts/2019-04-05-14th-dotty-milestone-release.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.14.0-RC1 with export, immutable arrays, creator applications and more
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2019-04-05
+---
+
+Hello! This is the 14th release of Dotty. Some of the most interesting changes in this release include the new `export`, the dual of `import`, feature, an immutable array type and the creator applications syntax.
+
+This release serves as a technology preview that demonstrates new
+language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are being considered for
+inclusion in Scala 3. Scala has pioneered the fusion of object-oriented and
+functional programming in a typed setting. Scala 3 will be a big step towards
+realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs together well,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 14th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+# What’s new in the 0.14.0-RC1 technology preview?
+
+## Export as a dual of Import
+
+A new `export` keyword is added to the language that defines aliases for selected members of an object. Consider the following example:
+
+```scala
+class BitMap
+class InkJet
+class Printer {
+  type PrinterType
+  def print(bits: BitMap): Unit = ???
+  def status: List[String] = ???
+}
+class Scanner {
+  def scan(): BitMap = ???
+  def status: List[String] = ???
+}
+class Copier {
+  private val printUnit = new Printer { type PrinterType = InkJet }
+  private val scanUnit = new Scanner
+  export scanUnit.scan
+  export printUnit.{status => _, _}
+  def status: List[String] = printUnit.status ++ scanUnit.status
+}
+```
+
+When defined like this, the `Copier` class exposes the `scan` method of the `scanner` and all the methods of `printUnit` except the `status`. You can hence call them on the `Copier` as follows:

```

Comment:
exposes -> exports

Reply:
Hm, I believe "exports" is not the best choice here since we are trying to explain the `export` feature. The cyclical reference may be confusing. What do you think?

Reply:
Circularity between an english term and a dotty keyword 😆. Actually its not "exports" but "defines an alias". More accurate an on par with the documentation 👍

Reply:
Indeed :) Ok, I've updated it